resources/css/main.css

```diff
@@ -0,0 +1,128 @@
+* {
```

Comment:
we should be having more of a responsive aimed stylesheet

Reply:
ok